device/iot-device-samples/pnp-device-sample/TemperatureController/src/main/java/samples/com/microsoft/azure/sdk/iot/TemperatureController.java

```diff
@@ -0,0 +1,382 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package samples.com.microsoft.azure.sdk.iot;
+
+import com.microsoft.azure.sdk.iot.deps.twin.TwinCollection;
+import com.microsoft.azure.sdk.iot.device.*;
+import com.microsoft.azure.sdk.iot.device.DeviceTwin.*;
+import com.microsoft.azure.sdk.iot.pnphelpers.PnpHelper;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.*;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static java.util.Map.*;
+
+@Slf4j
+public class TemperatureController {
+
+    public enum StatusCode {
+        COMPLETED (200),
+        IN_PROGRESS (202),
+        NOT_FOUND (404);
+
+        private final int value;
+        StatusCode(int value) {
+            this.value = value;
+        }
+    }
+
+    // DTDL interface used: https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/TemperatureController.json
+    private static final String deviceConnectionString = System.getenv("IOTHUB_DEVICE_CONNECTION_STRING");
+    private static final String MODEL_ID = "dtmi:com:example:TemperatureController;1";
+    private static final String THERMOSTAT_1 = "thermostat1";
+    private static final String THERMOSTAT_2 = "thermostat2";
+    private static final String SERIAL_NO = "SR-123456";
```

Comment:
call it Property to update? or add a comment?

Reply:
This value can ideally be used anywhere, property update/ telemetry/ command response etc. I don't see the value in tying it to the usage here.